homeassistant/components/cover/zwave.py

```diff
@@ -91,19 +91,28 @@ def current_cover_position(self):
 
     def open_cover(self, **kwargs):
         """Move the roller shutter up."""
-        self._network.manager.pressButton(self._open_id)
+        if not self._open_id:
+            self.node.set_dimmer(self.values.primary.value_id, 100)
+        else:
+            self._network.manager.pressButton(self._open_id)
 
     def close_cover(self, **kwargs):
         """Move the roller shutter down."""
-        self._network.manager.pressButton(self._close_id)
+        if not self._close_id:
+            self.node.set_dimmer(self.values.primary.value_id, 0)
+        else:
+            self._network.manager.pressButton(self._close_id)
 
     def set_cover_position(self, position, **kwargs):
         """Move the roller shutter to a specific position."""
         self.node.set_dimmer(self.values.primary.value_id, position)
 
     def stop_cover(self, **kwargs):
         """Stop the roller shutter."""
-        self._network.manager.releaseButton(self._open_id)
+        if not self._open_id:
+            self._network.manager.releaseButton(self.values.primary.value_id)
```

Comment:
You are right, looking at the xml from the user, the level does not respond to button presses. That means we need to bring in for either "Dim" or "Bright" value which is a button.  These will probably not be present with devices that do have open and close id's. We really need someone with a fibaro fgrm unit to help us test this.